internals/feature_helpers.py

```diff
@@ -90,21 +91,35 @@ def get_features_in_release_notes(milestone: int):
           Stage.milestones.webview_last >= milestone,
           Stage.rollout_milestone >= milestone),
       Stage.stage_type.IN([STAGE_BLINK_SHIPPING, STAGE_PSA_SHIPPING,
-          STAGE_FAST_SHIPPING, STAGE_DEP_SHIPPING, STAGE_ENT_ROLLOUT])).filter().fetch()
+          STAGE_FAST_SHIPPING, STAGE_DEP_SHIPPING, STAGE_ENT_ROLLOUT])).fetch()
+  logging.info('Fetched %r relevent stages', len(stages))
+
+  feature_ids = list({s.feature_id for s in stages})
```

Comment:
Is there a chance for duplicates?

I see before you used a set:

```
  feature_ids = list(set({
      *[s.feature_id for s in stages]}))
 ```

Reply:
The `{...}` does create a set.  The old code was actually creating two sets and two lists.